lms/static/sass/course/modules/_student-notes.scss

```diff
@@ -68,8 +101,190 @@
+      background-position: 0 0;
+    }
+  }
+}
+
+// editing
+.annotator-editor {
+
+  .annotator-controls {
+    @include text-align(left);
+    @include clearfix();
+    background: $notes-annotator-background-med !important; //matches annotator JS editing bubble triangle color
+    font-family: $f-sans-serif !important;
+    padding: 8px;
+    border: none !important;
+    border-radius: 0 !important;
+
+    // actions
+    .annotator-save, .annotator-cancel {
+      @extend %notes-reset-background;
+      font-family: $f-sans-serif !important;
+      font-size: 14px !important;
+      padding: ($baseline/4) ($baseline/2) !important;
+      border: none;
+      box-shadow: none;
+      text-shadow: none !important;
+
+      // removing vendor icons
+      &:after {
+        display: none !important;
+      }
+    }
+
+    .annotator-save {
+      @include float(left);
+    }
+
+    .annotator-cancel {
+      background-color: $transparent !important;
+    }
+  }
+
+  .annotator-item {
+
+    textarea {
+      @extend %notes-reset-font;
+      @extend %t-demi-strong;
+      padding: ($baseline/5) !important;
+      font-size: 14px !important;
+      line-height: 22px !important;
+      color: $gray-d3 !important;
+      background: $notes-annotator-background-light !important; //matches annotator JS editing bubble triangle color
+
+      // STATE: hover/focus
+      &:hover, &:focus {
+        background: $notes-annotator-background-light;
+      }
+    }
+  }
+}
+
+
+// +listing notes (overrides for vendor styling)
+// --------------------
+// highlight
+.annotator-hl {
+  background: $student-notes-highlight-color-focus;
+}
+
+// content
+.annotator-viewer {
+
+  // poorly scoped selector for content of a note's comment
+  div:first-of-type {
+    @extend %notes-reset-font;
+    padding: ($baseline/4) !important;
+    font-size: 14px !important;
+    line-height: 22px !important;
+    color: $gray-d2 !important;
+  }
+
+  // controls
+  .annotator-controls {
+    // RTL support
+    @include right(0);
+    top: 0;
+    @include float(right);
+    @include padding-left($baseline/4);
+
+    .annotator-delete, .annotator-edit {
+      position: relative;
+      display: inline-block;
+      vertical-align: middle;
+
+      &:before {
+        @extend %notes-reset-icon;
+        @extend %shame-link-base;
+        @extend %t-icon4;
+        position: absolute;
+      }
+    }
+
+    .annotator-edit {
+      @include margin-right($baseline/2);
+
+      &:before {
+        top: 0;
+        @include left(-($baseline/4));
+        content: "\f044";
+      }
+    }
+
+    .annotator-delete {
+
+      &:before {
+        top: 0;
+        @include left(-($baseline/4));
+        content: "\f00d";
```

Comment:
NOTE: Injecting Font Awesome glyphs in pseudo elements to make controls look consistent with LMS iconography and UI styling.